styles.less

```diff
@@ -323,7 +323,32 @@ body {
 		}
 
 		.slide-7 {
-			
+      // Previously at the end of the LESS file
+      .slide-7-dark-cloud {
+        position: absolute;
+        width: 15%;
+        height: 15%;
+        left: 74vw;
+        bottom: 78vh;
+        transition: 300ms 500ms;
+        /* Waiting to apply until cloud background is removed
+        -webkit-box-shadow: 9px 9px 7px 0px rgba(50, 50, 50, 0.75);
+        -moz-box-shadow:    9px 9px 7px 0px rgba(50, 50, 50, 0.75);
+        box-shadow:         9px 9px 7px 0px rgba(50, 50, 50, 0.75);
```

Comment:
box-shadow will actually always apply a shadow to the image as if it's a square. The way to achieve the affect in the mockups is to use a drop-shadow filter, described in this post (https://stackoverflow.com/questions/3186688/drop-shadow-for-png-image-in-css). We've tried this a bit using the mix-in .drop-shadow() that is currently in the LESS file, but it was causing performance problems, so we removed it. Let's leave the clouds without drop shadows for now.